src/components/TrackersManager.vue

```diff
@@ -0,0 +1,309 @@
+      <div class="col mb-2">
+        <h5>Already Filed</h5>
+        <div v-if="isLoadingTrackers" class="ms-2 mb-1">
+          <span
+            class="spinner-border spinner-border-sm"
+            role="status"
+          >
+            <span class="visually-hidden">Loading...</span>
+          </span>
+          <span> Loading trackers&hellip; </span>
+        </div>
+        <div v-if="alreadyFiledTrackers?.length === 0 && !isLoadingTrackers" class="ms-2">
+          <span> No filed trackers&hellip; </span>
+        </div>
+        <div v-else-if="!isLoadingTrackers" class="osim-tracker-list">
+          <label
+            v-for="(tracker, index) in alreadyFiledTrackers"
+            :key="`${tracker.ps_update_stream}:${tracker.ps_component}:${index}`"
+          >
+            <input
+              checked
+              disabled
+              type="checkbox"
+              class="osim-tracker form-check-input ms-2"
+            />
+            {{ `${tracker.ps_update_stream} (${tracker.ps_component})` }}
+          </label>
+        </div>
+      </div>
+      <div class="col">
+        <div>
+          <h5>Untrackable Affects
+            <i
+              class="bi bi-exclamation-circle text-danger"
+              :title="'These affects do not have available trackers.'
+                +' This may indicate an issue with product defintions.'
+                +' Please contact the OSIDB/OSIM team for assistance.'"
+            />
+          </h5>
+          <div v-if="isLoadingTrackers" class="ms-2 mb-1">
+            <span
+              class="spinner-border spinner-border-sm"
+              role="status"
+            >
+              <span class="visually-hidden">Loading...</span>
+            </span>
+            <span> Loading trackers&hellip; </span>
+          </div>
+          <div v-if="untrackableAffects?.length === 0 && !isLoadingTrackers" class="ms-2">
+            <span> No untrackable affects&hellip; </span>
+          </div>
+          <div
+            v-if="!isLoadingTrackers && untrackableAffects?.length > 0"
+            class="osim-tracker-list"
+          >
+            <span v-for="affect in untrackableAffects" :key="`${affect.ps_module}-${affect.ps_component}`">
+              <span class="ps-2 text-danger">
+                <i class="bi bi-exclamation-triangle"></i>
+                {{ affect.ps_module }}/{{ affect.ps_component }}
+              </span>
+            </span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <button
+      v-osim-loading.grow="isFilingTrackers"
+      type="button"
+      class="btn btn-sm btn-black osim-file-trackers mt-4"
+      :disabled="!trackersToFile.length || isFilingTrackers || isLoadingTrackers"
+      @click="handleFileTrackers"
+    >
+      <i v-if="!isFilingTrackers" class="bi bi-archive"></i>
+      File Selected Trackers
+    </button>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+@import '@/scss/bootstrap-overrides';
```

Comment:
This is already imported in the index, do we need it here? https://github.com/RedHatProductSecurity/osim/blob/bb264759f1c7e4fc460e5d6d7e682a2974a7b325/src/scss/index.scss#L1

Reply:
May not be needed here since no sass variables are used here